src/metabase/db/custom_migrations/metrics_v2.clj

```diff
@@ -0,0 +1,158 @@
+(ns metabase.db.custom-migrations.metrics-v2
+  (:require
+   [cheshire.core :as json]
+   [metabase.util :as u]
+   [metabase.util.log :as log]
+   [toucan2.core :as t2])
+  (:import (java.time Instant)))
+
+(set! *warn-on-reflection* true)
+
+(defn- get-metric-migration-collection
```

Comment:
Based on https://stuartsierra.com/2016/01/09/how-to-name-clojure-functions I think you should just call this `metric-migration-collection`

Reply:
Actually maybe `get-or-create-metric-migration-collection!` since it's (potentially) destructive

Reply:
As far as I can tell you only ever use the ID of this Collection (everything else about the Collection is irrelevant) so maybe this should actually be `get-or-create-metric-migration-collection-id!` and return an ID, then you can do `select-one-pk` and `insert-returning-pks!` instead

Reply:
Good catch. You were looking at a piece of code from a much more complicated version of this migration. I'll see what I can do.

Reply:
Actually, the name of the collection was used in one place, but it's well known, and maybe even not that important.